site/content/extend/plugins/community-plugin-marketplace.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Community Plugins in the Marketplace
+subsection: Plugins (Beta)
+weight: 60
+---
+
+Once your plugin has reached a certain level of quality, you might consider submitting it to the Plugin Marketplace. The Plugin Marketplace allows discovery, installation and updates of plugins directly within Mattermost. It is a great way to get feedback on your plugin and help make it more popular. Once your plugin is accepted to the Marketplace, Mattermost will also send you swag!
+
+### Requirements for adding community plugin to the Marketplace
+Every community plugin must fulfill the following checklist to be added to the Marketplace:
+
+**Product Requirements (Checked by a Product Manager)**
+
+1. The plugin is published under an Apache v2 compatible license (e.g. no GPL, APGL). A list of compatible licenses can be found [here](https://apache.org/legal/resolved.html#category-a).
+2. The source code is available in a public git repository.
+3. There is a public issue or bug tracker for the plugin, which is linked in the plugin documentation and linked via `support_url` in the manifest.
+4. The plugin provides detailed usage documentation with at least one screenshot of the plugin in action, list of features and a development guide. This is typically a README file or a landing page on the web. The link to the documentation is set as `homepage_url` in the manifest. A great example is the [README of the GitHub plugin](https://github.com/mattermost/mattermost-plugin-github/blob/master/README.md).
```

Comment:
This would cleaner as:
```markdown
A great example is the [the GitHub plugin's README](https://github.com/mattermost/mattermost-plugin-github/blob/master/README.md).
```
or:
```markdown
[The GitHub plugin's README](https://github.com/mattermost/mattermost-plugin-github/blob/master/README.md) is a great example.
```

Reply:
I'm not a native speaker. Could you please explain what the reasoning is behind using `the GitHub plugin's README` over `README of the GitHub plugin`?

Reply:
It’s fairly minor and likely a matter of opinion, but I think that it reads more cleanly. Saying `the GitHub plugin’s README` keeps it as a singular concept: the README that is part of the GitHub plugin. Saying `the README of the GitHub plugin` is slightly more verbose and also requires the reader to associate two distinct concepts: some README and the GitHub plugin. In this case, that’s fairly simple and automatic, but it can be more complicated depending on the rest of the sentence.